packages/account/src/Components/Routes/route-with-sub-routes.jsx

```diff
@@ -18,9 +19,15 @@ const RouteWithSubRoutes = route => {
         } else if (route.is_authenticated && !route.is_logged_in && !route.is_logging_in) {
             redirectToLogin(route.is_logged_in, getLanguage());
         } else {
-            const default_subroute = route.routes ? route.routes.find(r => r.default) : {};
+            const sub_paths = [];
+            if (route.routes) {
+                route.routes.map(r => r.subroutes.forEach(p => sub_paths.push(p)));
+            }
```

Comment:
IMO using `map` is not necessary here. purpose of `map` is **generating** a new array from another array.
here you are not assigning or using the result of `map` so `map` is not necessary.
you can use `forEach` instead, but as a suggestion, I think the best way for generating `sub_path` is by using `reduce`:

```suggestion
            const sub_paths = (route.routes ?? []).reduce((acc, cur) => {
                return acc.concat(cur.subroutes); 
            }, []);
```